application/models/UserDatabase.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+class UserDatabase {
+
+  protected $connection;
+
+  public function __construct() {
+    $this->connection = new mysqli(SERVER, USER, PASS, DBNAME);
+    // Check connection
+    if ($this->connection->connect_error) {
+      die("Connection failed: " . $conn->connect_error);
+    }
+  }
+
+  public function insertUserInfo($name, $email, $dob, $gender, $pass) {
+    $sql = "insert into user(name, email, dob, gender, passcode) 
```

Comment:
Whenver you accept any value from user, you need to sanitize the values so that user does not do sql injection.